reports/portland

```diff
@@ -247,7 +247,7 @@ tags: less-lethal, projectile, protester, shoot
 
 id: or-portland-9
 
-geolocation: 
+geolocation: 45.514871, -122.680014 // Unitus Plaza caught on video
```

Comment:
Instead of commenting here, can you add a note in the description section?

Reply:
Oh I may have been unclear. I meant that if you could include the note in the description section under the incident title, but this is also fine.

The real thing is I'm not sure the data formatters would have a good time with the comment format.

If you can resolve merge conflicts, then I'll approve

Reply:
For ref, this is what I normally do: https://github.com/2020PB/police-brutality/pull/2012/files#diff-02f181ba32920e51f215d4c258a6171ec3d9dc49883e505a55114a38d840994bL6309